src/System.Private.CoreLib/shared/System/Runtime/MemoryFailPoint.cs

```diff
@@ -245,22 +238,13 @@ LastKnownFreeAddressSpace < (long)segmentSize)
                         if (!needPageFile)
                             continue;
 
+                        // Attempt to grow the OS's page file.  Note that we ignore
+                        // any allocation routines from the host intentionally.
+                        RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
Nit: These `PrepareConstrainedRegions` calls are no-ops in .NET Core. They do not hurt, but we have been deleting them.

Reply:
I didn't delete it because it was not empty method on CoreCLR. It just calls an empty `ProbeForSufficientStack` method, but there was some comment around it that seemed to suggest that "magic" can happen.